1. Minor change: Removal of "*Nearly all natural HEGs are embedded in self-splicing introns/inteins, help to min. disruption of target.*" on p.1
2. Minor change: "*versatility of gene drives is constrained by difficulty of retargeting enzymes...*" changed to just "*gene drive are constrained...*"
3. "*The question of whether Cas9 can be used to build efficient gene drives is highly relevant due to the potential for such constructs to alter any gene in any population that can be edited with Cas9, a group that may constitute every species tested to date*" changed to "*Because Cas9 can apparently be used to edit any gene, the question of whether it can be used to gene drives capable of spreading those changes through wild populations is highly relevant.*" on p.2
4. "*whether Cas9 can bias inheritance in a eukaryotic organism at all remains unknown*" changed to "*whether Cas9 can bias inheritance at all remains unknown*" on p.2
5. "*whether initiating public discussions and engaging in regulatory reform is truly necessary.*" changed to "*whether initiating public discussions and engaging in regulatory reform are immediately necessary*" on p.2
6. Detail removed from end of Introduction in favour of "*constructing several types of RNA-guided gene drives and quantifying their ability to bias inheritance*".

### Results section

1. Removed: "*The inheritance-biasing efficiency of an endonuclease gene drive is determined by 1) the frequency of cutting and 2) the fraction of repair events that lead to the drive being copied onto the target chromosome. Only repair by homologous recombination (HR) results in drive copying and inheritance bias; the competing non-homologous end-joining (NHEJ) repair pathway ligates the broken ends together, preventing the drive from being copied and often generating mutations at the site of the break that prevent further cutting*" on p.2


2. Emphasised stringency of precautions on p.2



3. "*This represents the first example of a synthetic endonuclease gene drive that biases its own inheritance when mated to a wild-type eukaryote*" changed to "*At the time of submission and preprint release, this was the first example of a synthetic endonuclease gene drive that biases its own inheritance when mated to a wild-type organism.*" on p.4.



4. Explanation of recording with function retention reduced from paragraph to sentence. "*While both ADE2-based gene drives are highly efficient at biasing inheritance in the laboratory, they would be unable to cut a chromosome lacking the non-essential ADE2 gene or one with a mutation in the targeted sequence. This is a particular problem for drives seeking to alter entire populations because any aberrant repair by nonhomologous end-joining (NHEJ) following cutting will remove the target sites and thereby generate a driveresistant allele. We previously postulated that gene drives that target and recode an essential gene could avoid drive resistance in large populations because error prone repair events that modify the target site would cause lethality1 . Non-essential but nonetheless important genes could similarly be edited because mutants created by NHEJ events would still be less fit than the drive itself.*" changed to "*We subsequently sought to determine whether gene drives can not simply disrupt a gene, but recode it and leave the function intact.*"

5. "*indicating a continued ability to spread through sexually reproducing populations over multiple generations so long as the fitness cost associated with the drive is less than the inheritance benefit in the relevant environment.*" removed, instead added that:
  
  <hr id="added-drive">
  
  "*We designed the previously described gene drives to be incapable of autonomous spread due to their requirement for exogenously supplied Cas9 (Fig. 2C-D, Fig. 4). To determine whether an autonomous drive encoding the large cas9 gene is similarly efficient, we constructed one such drive targeting a recoded synthetic sequence within ADE2 (Fig. 5A). We further sought to determine whether the loss of ADE2 function induced by this drive might be undone using a “reversal drive”. We consequently built a second autonomous gene drive to cleave the first autonomous ADE2-disrupting drive and subsequently restore an intact copy of the ADE2 gene (Fig. 5B). Quantitative PCR demonstrated that both of these autonomous drives were copied at an average efficiency over 99% (Fig. 5D). Together, these results demonstrate the efficacy and reversibility of autonomous Cas9- based gene drives.*" on p.6
